sounds/models.py

```diff
@@ -169,7 +169,7 @@ def bulk_query_solr(self, sound_ids):
         FROM
           sounds_sound sound
           LEFT JOIN auth_user ON auth_user.id = sound.user_id
-          LEFT JOIN sounds_pack ON sound.pack_id = sounds_pack.id
+          LEFT JOIN sounds_pack ON (sound.pack_id = sounds_pack.id AND sounds_pack.is_deleted = False)
```

Comment:
I did a quick test with a pack marked as deleted and I get the same results adding the extra `AND sounds_pack.is_deleted = False` than when I don't add it (which is `pack_id` and `pack_name` being null in the query). If I understand all of this correctly, this means that we are not really indexing deleted packs because when we delete a pack, the foreign key in sound is set to null.

However, if for some reason the foreign key is not set to null and the sound still points to pack X, then this extra clause will set the `pack_name` prop to null when `pack_id` points to a (deleted) pack. In that case we need the improvement in this PR to avoid indexing the deleted pack. Hence, this improvement fixes the problem when the data is not fully consistent in the DB, but does not look at the root problem which is to understand why we're getting sounds pointing to deleted packs (if any). In fact, I just checked now in production DB and running `Sound.objects.filter(pack__is_deleted=True).count()` returns 7 sounds, and these are 6 sounds from pack 20169 and 1 sound from pack 20836.

I'm now sure how to proceed, but I'm inclined to think that we should not merge this patch because it will potentially hide the problem of sounds pointing to deleted packs. I'd fix the ones which are wrong in DB now, and look again in this problem only if it reappears in the future.

Reply:
I never though about that, I guess at some point we could remove the reference between the sound and the deleted pack, but then the question is why we keep the deleted pack?.

Reply:
I don't remember why we keep it, maybe to show the "deleted pack" page. The reference between the sound and the deleted pack is supposed to be removed, that is fine.

Reply:
I checked and we keep the reference to show the pack as deleted. This is fine.
As mentioned above, this fix only solves the problem in a very specific case. I'll do the following:
 * Fix current sounds pointing to deleted packs and reindex these sounds
 * Close this PR without merging
 * Check in some time from now (i.e. six months) if the problem appeared again

If the problem appears again then we further investigate the causes and (possibly) apply this PR plus some other required changes.